scripts/regenerate.sh

```diff
@@ -19,12 +19,16 @@ WORKDIR=$(mktemp -d)
 
 function finish {
   rm -rf "$WORKDIR"
+  # Revert back the PATH to client's original value
+  export PATH=$ORIGINAL_PATH
+  echo "Restored PATH variable to original value."
```

Comment:
nit: remove this echo as well. The user doesnt have to know that the state changed and was reverted.

Reply:
Updated